BPA/Ch.06/Mh.stan

```diff
@@ -7,27 +7,27 @@ data {
 transformed data {
   int<lower=0> C;               // Size of observed data set
 
-  C <- 0;
+  C = 0;
   for (i in 1:M) {
     if (y[i] > 0)
-      C <- C + 1;
+      C = C + 1;
   }
 }
 
 parameters {
-  real<lower=0,upper=1> omega;  // Inclusion probability
-  real<lower=0,upper=1> mean_p; // Detection probability
-  real<lower=0> sigma;
-  real<lower=-16,upper=16> eps[M];      // Individual random effects
+  real<lower=0,upper=1> omega;      // Inclusion probability
+  real<lower=0,upper=1> mean_p;     // Mean detection probability
+  real<lower=0,upper=5> sigma;
```

Comment:
Yes, they are defined to match the model in the BPA book. I know that they are not recommended in Stan.